### PR TITLE
Metricbeat: Set descriptions for dashboards

### PR DESCRIPTION
**Pull request by jsoriano**

### CODE REVIEW

File: metricbeat/module/haproxy/_meta/kibana/6/dashboard/Metricbeat-haproxy-backend.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "objects": [
         {
             "attributes": {
-                "description": "", 
+                "description": "Haproxy backend metrics",
```
Comment (ruflin) [inline metricbeat/module/haproxy/_meta/kibana/6/dashboard/Metricbeat-haproxy-backend.json]: Should we call it `HAProxy`? Same below

Comment (jsoriano) [reply metricbeat/module/haproxy/_meta/kibana/6/dashboard/Metricbeat-haproxy-backend.json]: Yes, changed all appearances of HAProxy with different uppercasing.

**Comment by jsoriano** [comment]
Yes, much much better now to edit dashboards :smile:

File: metricbeat/module/haproxy/_meta/kibana/6/dashboard/Metricbeat-haproxy-http-server.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
+                "description": "HAProxy metrics for HTTP mode",
                 "hits": 0, 
                 "kibanaSavedObjectMeta": {
                     "searchSourceJSON": {
                         "filter": [], 
                         "highlightAll": true, 
                         "query": {
                             "language": "lucene", 
-                            "query": "haproxy.stat.service_name:\"nginx2\""
+                            "query": ""
```
Comment (ruflin) [inline metricbeat/module/haproxy/_meta/kibana/6/dashboard/Metricbeat-haproxy-http-server.json]: Good fix.

**Pull request merged**